src/scene_battle_rpg2k.cpp

```diff
@@ -15,6 +15,7 @@
  * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <iostream>
```

Comment:
> 2.22: RM2000: Correct printing of state heal/affect messages 

and this one

Reply:
Oops looks like I forgot to clean up my mess after debugging.